configure.sh

```diff
@@ -43,22 +44,28 @@ if [[ "${QUARTO_VENDOR_BINARIES}" = "true" ]]; then
 
     # Download Deno
     DENOURL=https://github.com/denoland/deno/releases/download
-    if [[ $OSTYPE == 'darwin'* ]]; then
-      DENOFILE=deno-x86_64-apple-darwin.zip
-    else
-      DENOFILE=deno-x86_64-unknown-linux-gnu.zip
-    fi
-    download "$DENOURL/$DENO/$DENOFILE" "$DENOFILE"
-    unzip -o $DENOFILE
-    rm $DENOFILE
+
+    for DENOFILE in $DENOFILES; do
+      download "$DENOURL/$DENO/$DENOFILE" "$DENOFILE"
+      unzip -o $DENOFILE
+      DENO_ARCH_DIR=$(basename $DENOFILE .zip)
+      mkdir $DENO_ARCH_DIR
+      mv deno $DENO_ARCH_DIR
+      rm $DENOFILE
+    done
+
+    ln -s $DENO_DIR/deno deno
```

Comment:
I don't fully fully understand the softlink here (don't other places in the code specifically look in the `DENO_DIR`?)

Reply:
This is simply to appease vs code and the workspace configuration file. I didn't want to use symlinks everywhere to err on the side of being explicit.